css/styles.css

```diff
@@ -1,63 +1,58 @@
-body{
+.container{
     width: 1280px;
-    height: 1626px;
+    margin: auto;
+    font-family: Roboto, sans-serif;
+}
+.container_nav{
```

Comment:
`.container_nav` is the incorrect name for this class, better to use just `.nav`